README.md

```diff
@@ -79,6 +80,10 @@ invoice: lnbcrt100n1pjuvtdpp...
 
 `./nutw receive cashuAeyJ0b2tlbiI6W3...`
 
+### pay an invoice
+
+'./nutw pay lnbc100n1pju35fedqqsp52xt3...`
```

Comment:
Thanks. Could you fix formatting here (backtick)

Reply:
Fixed, sorry.